examples/backtest.ipynb

```diff
@@ -23,15 +23,33 @@
     "\n",
```

Comment:
<p>Create a separate subsection for backtest with fold masks and add the examples here how to implement the strategies form <a href="https://github.com/tinkoff-ai/etna/issues/983" target="_blank">here</a> using fold masks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>